server.ts

```diff
@@ -20,17 +21,49 @@ const dbConfig = {
 
 const app = express();
 
-app.use(express.json()); //add body parser to each following route handler
+app.use(express.json()); //add body parser to each following route handler and allows us to have access to request.body
 app.use(cors()) //add CORS support to each following route handler
 
 const client = new Client(dbConfig);
 client.connect();
 
-app.get("/", async (req, res) => {
-  const dbres = await client.query('select * from categories');
-  res.json(dbres.rows);
+app.get("/pastes/tenPastes", async (req, res) => {
+  try {
+    const allPastes = await client.query('select * from pastebin order by time desc limit 10');
+    res.json(allPastes.rows);
+  } catch (error) {
+    console.error(error.message)
+  }
+});
+
+app.get("/pastes", async (req, res) => {
+  try {
+    const allPastes = await client.query('select * from pastebin');
+    res.json(allPastes.rows);
+  } catch (error) {
+    console.error(error.message)
+  }
 });
 
+app.get('/pastes/:id',(req,res)=>{  
+  client.query('SELECT * FROM pastebin WHERE entry_id = $1',[req.params.id],(err,rows)=>{  
+  if(!err)   
```

Comment:
a bit unclear on why you chose this syntax - maybe try/catch syntax would be more comprehensive for error catching